src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -56,9 +63,16 @@ public function __construct(
         private string $firewallName,
         private ?LoggerInterface $logger = null,
         private bool $eraseCredentials = true,
-        private bool $hideUserNotFoundExceptions = true,
+        string|bool $exposeSecurityErrors = self::EXPOSE_ACCOUNT_STATUS,
```

Comment:
the default value needs to be `EXPOSE_NONE`, to not change the existing behavior.